### DISCUSSION TITLE
How to capture metadata from Dagster

### DISCUSSION THREAD

**Discussion by nehiljain** [recap-build/recap#191] [Q&A] [answered]
It would be amazing if I can centralize metadata of dag runs inside Dagster and lineage from Dbt on snowflake in one repository.

[Dagster IO Metadata API](https://docs.dagster.io/concepts/io-management/io-managers#recording-metadata-from-an-io-manager)

[dbt Metadata API](https://docs.getdbt.com/docs/dbt-cloud-apis/metadata-api)

Then I can start doing some analysis in my CICD pipelines to prevent failures from breaking changes between consumers and producers. Eventually enforcing data contracts.

**Comment by criccomini** [comment]
Going to convert this to a GH issue.

**Comment by criccomini** [comment, marked as answer]
So it sounds like there are a few things here:

1. Dagster jobs in Recap
2. dbt job lineage in Recap

Does that sound right?

**Reply by nehiljain** [reply]
Yep, these two would be analyzers for the two.
